examples/spark_udf/spark_udf_with_prebuilt_env.py

```diff
@@ -0,0 +1,41 @@
+"""
+This example code shows how to use `mlflow.pyfunc.spark_udf` with Databricks Connect
+outside Databricks runtime.
+"""
+
+import os
+
+from databricks.connect import DatabricksSession
+from sklearn import datasets
+from sklearn.neighbors import KNeighborsClassifier
+
+import mlflow
+
+spark = DatabricksSession.builder.remote(
+    host=os.environ["DATABRICKS_HOST"],
+    token=os.environ["DATABRICKS_TOKEN"],
+    cluster_id="<cluster id>",  # get cluster id by spark.conf.get("spark.databricks.clusterUsageTags.clusterId")
+).getOrCreate()
+
+X, y = datasets.load_iris(as_frame=True, return_X_y=True)
+model = KNeighborsClassifier()
+model.fit(X, y)
+
+with mlflow.start_run():
+    model_info = mlflow.sklearn.log_model(model, "model")
+
+model_uri = model_info.model_uri
+
+# The prebuilt model environment archive file path.
+# To build the model environment, run `mlflow.pyfunc.build_model_env` in Databricks runtime,
+# and then download the built env archive file to local machine.
```

Comment:
If `mlflow.pyfunc.build_model_env` must be called with a VC volume, that should be mentioned.